beta/src/components/Layout/Sidebar/Sidebar.tsx

```diff
@@ -18,7 +18,7 @@ export function Sidebar({isMobileOnly}: {isMobileOnly?: boolean}) {
   const {menuRef, isOpen} = React.useContext(MenuContext);
   const isMobileSidebar = useMediaQuery(SIDEBAR_BREAKPOINT);
   let routeTree = React.useContext(SidebarContext);
-  const isHidden = isMobileOnly && !isMobileSidebar;
+  const isHidden = isMobileSidebar ? !isOpen : 'false';
```

Comment:
This is confusing that it’s a string. Let’s keep the variable Boolean and do the string thing directly in the aria prop.